pages/cart-management.html

```diff
@@ -0,0 +1,277 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Meta Information -->
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Roshan J" />
+    <meta
+      name="description"
+      content="Gentle UI is a component library having pre built components with the motive of creating good UIs."
+    />
+
+    <!-- Custom Favicon -->
+    <link
+      rel="icon"
+      href="https://img.icons8.com/color/64/000000/boy-stroller.png"
+    />
+
+    <!-- Fonts and icons stylesheets -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Amatic+SC:wght@400;700&family=Roboto+Slab:wght@300;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
+
+    <!-- Stylesheets attached -->
+    <link
+      rel="stylesheet"
+      href="https://deploy-preview-12--gentle-ui.netlify.app/gentle-imports.css"
+    />
+    <!-- Main stylesheet added -->
+    <link rel="stylesheet" href="../styles/cart-management.css" />
+    <title>Cart Management</title>
+  </head>
+  <body>
+    <div class="cart-management-layout">
+      <!-- Navigation begins -->
+      <nav class="gentle-navbar chillar-ecom-nav navbar-component">
+        <div class="brand">
+          <a href="../index.html" class="link-none">
+            <img
+              loading="lazy"
+              src="../assets/images/brand-logo.png"
+              alt="brand description"
+              class="brand-icon"
+            />
+          </a>
+        </div>
+        <div class="gentle-search-group gentle-navbar-search">
+          <i class="fas fa-search search-icon"></i>
+          <input
+            type="search"
+            class="gentle-search-bar"
+            placeholder="Search-for-products"
+          />
+        </div>
+        <ul class="gentle-nav-links ecom-nav-links">
+          <li>
+            <a class="link-none" href=""> LOGIN </a>
+          </li>
+          <li>
+            <a class="link-none" href="cart-management.html">
+              <div class="badge mx-2">
+                <i class="fas fa-cart-plus fa-2x"></i>
+                <div class="badge-number gentle-flex-center">
+                  <span>2</span>
+                </div>
+              </div>
+            </a>
+          </li>
+          <li>
+            <a class="link-none" href="">
+              <div class="badge mx-2">
+                <i class="fas fa-heart fa-2x"></i>
+                <div class="badge-number gentle-flex-center">
+                  <span>2</span>
```

Comment:
`span` is not necessary to add here as you are not adding any style to it

Reply:
Sure would implement these changes!!